app/views/games/index.html.erb

```diff
@@ -1 +1,165 @@
-<h1>StoicChess</h1>
\ No newline at end of file
+<br />
+<section id="register">
+  <div class="col-sm-5 col-sm-offset-1">
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <br />
+  </div>
+  <div class="col-sm-3 col-sm-offset-1">
+    <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="form-inputs">
+          <br />
+          <br />
+          <br />
+          <%= f.input :email, required: false, autofocus: true %>
+          <br />
+          <%= f.input :password, required: false %>
+          <br />
+          </div>
+          <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
+          <%= render "devise/shared/forgotpassword" %>
+          <br />
+        <div class="form-inputs">
+          <%= f.button :submit, "LOG IN", class: 'btn btn-success login' %>
+        </div>
+    <% end %>
+  </div>
+</section>
+<br class="clr" />
+<hr class="col-sm-offset-1 col-sm-10"/>
```

Comment:
You shouldn't use columns without rows. This breaks the nesting and will flow all of your columns into each other